packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+import { FzColumn, FzColumnSlots, FzColumnProps} from '@fiscozen/simple-table'
+import { FzIconDropdown } from '@fiscozen/dropdown';
+import { FzButton } from '@fiscozen/button';
+
+const props = withDefaults(defineProps<FzTableProps>(), {
+  pageInterval: 2,
+  activePage: 0
+})
+
+const emit = defineEmits([])
+const activePage = defineModel<number>('activePage')
+
+const slots = useSlots();
+
+const defaultSlot = slots.default?.();
+const columns =
+  defaultSlot
+    ?.filter((elem) => elem.type === FzColumn)
+    .map((column) => ({
+      props: column.props as FzColumnProps,
+      children: column.children as FzColumnSlots,
+    })) ?? [];
+
+const grid = ref()
+
+const staticClasses = ref(['grid'])
+const classes = computed(() => {
+  let res: string[] = [];
+
+  const colLength = props.actions ? columns.length + 1 : columns.length
+
+  switch (colLength) {
+    case 1:
+      res.push('grid-cols-1') 
+      break;
+    case 2:
+      res.push('grid-cols-2') 
+      break;
+    case 3:
+      res.push('grid-cols-3') 
+      break;
+    case 4:
+      res.push('grid-cols-4') 
+      break;
+    case 5:
+      res.push('grid-cols-5') 
+      break;
+    case 6:
+      res.push('grid-cols-6') 
+      break;
+    case 7:
+      res.push('grid-cols-7') 
+      break;
+    case 8:
+      res.push('grid-cols-8') 
+      break;
+    case 9:
+      res.push('grid-cols-9') 
+      break;
+    case 10:
+      res.push('grid-cols-10') 
+      break;
+    case 11:
+      res.push('grid-cols-11') 
+      break;
+    case 12:
+      res.push('grid-cols-12') 
+      break;
+    default:
+      break;
+  }
+  return res;
+})
+
+const colspan = computed(() => {
+  let res: string[] = [];
+
+  const colLength = props.actions ? columns.length + 1 : columns.length
+
+  switch (colLength) {
+    case 1:
+      res.push('col-span-1') 
+      break;
+    case 2:
+      res.push('col-span-2') 
+      break;
+    case 3:
+      res.push('col-span-3') 
+      break;
+    case 4:
+      res.push('col-span-4') 
+      break;
+    case 5:
+      res.push('col-span-5') 
+      break;
+    case 6:
+      res.push('col-span-6') 
+      break;
+    case 7:
+      res.push('col-span-7') 
+      break;
+    case 8:
+      res.push('col-span-8') 
+      break;
+    case 9:
+      res.push('col-span-9') 
+      break;
+    case 10:
+      res.push('col-span-10') 
+      break;
+    case 11:
+      res.push('col-span-11') 
+      break;
+    case 12:
+      res.push('col-span-12') 
+      break;
+    default:
+      break;
+  }
+  return res;
+})
+
+const headerStaticClasses = ['fz__header__cell','sticky','top-0','z-[2]', 'bg-grey-100', 'px-16', 'h-48', 'font-medium', 'flex', 'justify-start', 'items-center']
+const headerClasses = computed(() => {
+
+})
+
+const bodyStaticClasses = ['fz__body', 'z-[1]', 'px-16', 'h-48', 'bg-core-white', 'flex', 'justify-start', 'items-center', 'min-w-min', 'border-b-1', 'border-grey-100']
+const bodyClasses = computed(() => {
+
+})
```

Comment:
empty?
I'd just make a single computed for each element, I don't see the need to separate between static and non static

Reply:
Wasn't this agreed before to separate static from dynamic? of course I will remove empty ones